cablelabs/lpwanserver
# Issue 48b

#### What does this PR do?

Fix double network creation when user continues to edit after bad auth, and then resubmits
Shore up state management when user continues to edit after bad auth
Improve naming/organization of modules 

#### Do you have any concerns with this PR?

Some console.log statements are present to facilitate testing, should be removed after testing done

#### How can the reviewer verify this PR?

Create NW with bad auth info, "Continue Editing", fix auth info and submit.  Should work, and only create single network

#### Any background context you want to provide?
No

#### Screenshots or logs (if appropriate)

#### Questions:
- Have you connected this PR to the issue it resolves?
#42 

- Does the documentation need an update?
yes

- Does this add new dependencies?
no
- Have you added unit or functional tests for this PR?
no
